Update ESdd007.xml

MagdaKrzyz opened the pull request:
I had to make some changes in the msContent. I have deleted some roles if they were stated between the encoding.

thea-m reviewed (approved):
EVerything is fine, but for my understanding, could you explain to me the improvement of deleting the person roles from within the description?

MagdaKrzyz commented:
I deleted them where it is already said that it is a donor or an owner. 
Like here: 
![Role_name](https://github.com/user-attachments/assets/64b55b1e-951e-4ef6-a800-78b2b79cc795)

I added the list at the end so the persons will appear against the red background. 
I do not merge and wait for your response.

nafisa-valieva reviewed (approved):
Wonderful! Thank you! (I see your point about roles and wait to see what Dorothea replies)

thea-m commented:
It's fine, I was just curious :)
